task/04-arrays-tasks.js

```diff
@@ -559,7 +663,13 @@ function getElementByIndexes(arr, indexes) {
  * 
  */
 function swapHeadAndTail(arr) {
-    throw new Error('Not implemented');
+   const middle = Math.floor(arr.length / 2);
+
+   if(arr.length % 2 && arr.length > 1) {
+      arr.push(arr[middle]);
+      arr.splice(middle, 1);
```

Comment:
Never change arguments. It is a potential bug. Why? `splice` operator change `arr` and external code will work with mutated array.